### ISSUE TITLE
URGENT: Lockdown db.

### ISSUE THREAD

**Issue by nopivnick** [ITPNYU/mc-reservation-tool#299]
https://cloud.google.com/firestore/docs/security/get-started#auth-required

**Comment by rlho** [comment]
@nopivnick 
If we restrict the write settings, it will also prevent manually adding data to Firestore like we're doing now.

**Comment by nopivnick** [comment]
@rlho surely if we restrict the write settings to "authenticated users" we'd still be able to write manually, no?

**Comment by rlho** [comment]
@nopivnick yeah, we should use this function, could you design the rule?

**Comment by nopivnick** [comment]
@rlho yes, I will investigate and then coordinate with you and @lucia-gomez as far as when we implement so no one gets suprized if/when I inadvertently break something. removing you from the task.
